python/llm/example/GPU/LLM-Finetuning/axolotl/README.md

```diff
@@ -15,23 +15,24 @@ conda create -n llm python=3.11
 conda activate llm
 # below command will install intel_extension_for_pytorch==2.1.10+xpu as default
 pip install --pre --upgrade ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
-# install axolotl v0.3.0
+# install axolotl v0.4.0
 git clone https://github.com/OpenAccess-AI-Collective/axolotl
 cd axolotl
-git checkout v0.3.0
+git checkout v0.4.0
 cp ../requirements-xpu.txt requirements.txt
 pip install -e .
+pip install transformers==4.36.0
```

Comment:
maybe we could move this to `requirements-xpu.txt`?

Reply:
> maybe we could move this to `requirements-xpu.txt`?

No. This will cause error in repo scan.